src/libraries/System.Resources.ResourceManager/tests/TrimCompatibilityTests.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using Xunit;
+
+namespace System.Resources.Tests
+{
+    public static class TrimCompatibilityTests
+    {
+        /// <summary>
+        /// Verifies that ResourceReader.CreateUntypedDelegate doesn't have any DynamicallyAccessedMembers attributes,
+        /// so we can safely call MakeGenericMethod on its methods.
+        /// </summary>
+        [Fact]
+        public static void VerifyMethodsCalledWithMakeGenericMethod()
+        {
+            Assembly assembly = typeof(ResourceManager).Assembly;
+            Type type = assembly.GetType("System.Resources.ResourceReader");
```

Comment:
ResourceReader is public.

```suggestion
            Type type = typeof(ResourceReader);
```